website/project/model.py

```diff
@@ -3918,13 +3927,19 @@ def update_metadata(self, metadata):
         self.registration_metadata.update(metadata)
         return changes
 
+    def add_status_log(self, action):
+        self.status_logs.update({
+            datetime.datetime.utcnow().strftime('%Y-%m-%d %H:%M'): action
```

Comment:
Why are status logs represented as a dictionary keyed by date? Why optimize for accessing by a single date? It might make sense to follow the same modeling as NodeLog/Node.logs.